extensions/amp-analytics/0.1/vendors/deepbi.json

```diff
@@ -19,5 +19,13 @@
     "beacon": false,
     "xhrpost": false,
     "image": true
+  },
+  "linkers": {
+    "deepbi": {
+      "ids": {
+        "deepbiampid": "CLIENT_ID(deepbiamp)"
+      },
+      "proxyOnly": false
+    }
```

Comment:
Are you sure you want the linker to be disabled?

Reply:
I read in the documentation that it should be intentionaly turned on in the configuration but I would prefer to have this enabled by default. Would it be sufficient if I add:
`"enabled": true` 
?

Reply:
Yup this is the way to do it.